Parse/src/main/java/com/parse/ParseEventuallyQueue.java

```diff
@@ -30,6 +30,16 @@
    */
   private TestHelper testHelper;
 
+  private ParseHttpClient httpClient;
+
+  protected void setHttpClient(ParseHttpClient client) {
+    httpClient = client;
+  }
```

Comment:
is there any reason why we're using a getter and setter? why not use a constructor so we can make `httpClient` final?